dashboard/pages/relation_graph.tsx

```diff
@@ -98,42 +95,29 @@ export default function StreamingGraph() {
 
   // Periodically fetch fragment-level back-pressure from Meta node
   const [channelStats, setChannelStats] =
-    useState<Map<string, ChannelStatsDerived>>()
+    useState<Map<string, ChannelDeltaStats>>()
   const [relationStats, setRelationStats] = useState<{
     [key: number]: RelationStats
   }>()
 
   useEffect(() => {
-    // The initial snapshot is used to calculate the rate of back pressure
-    // It's not used to render the page directly, so we don't need to set it in the state
     let initialSnapshot: ChannelStatsSnapshot | undefined
 
     if (resetEmbeddedBackPressures) {
       setChannelStats(undefined)
       toggleResetEmbeddedBackPressures()
     }
 
-    function refresh() {
```

Comment:
Just reuse the refactored function above: https://github.com/risingwavelabs/risingwave/pull/22838/files#r2266136364